Week-3/memonize.js

```diff
@@ -0,0 +1,21 @@
+
+//closure
+const memoize = function (func) {
```

Comment:
@Dookiee JS 2 assignments looks good. Please complete the other 2 and let me know. Please fix the responsive issue which i mentioned. Thanks

Since all in one PR, i am not approving it.

Reply:
all JS problems for the week is completed. Please review and let me know!